docs/dxp/7.x/en/liferay-internals/extending-liferay/overriding-osgi-services/reconfiguring-components-to-use-your-osgi-service/resources/liferay-m1t1.zip/m1t1-impl/src/main/java/com/acme/m1t1/internal/M1T1Impl.java

```diff
@@ -0,0 +1,16 @@
+package com.acme.m1t1.internal;
+
+import com.acme.m1t1.M1T1;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(immediate = true, service = M1T1.class)
+public class M1T1Impl implements M1T1 {
+
+	@Override
+	public String doSomething() {
+		return this.getClass(
+		).getName();
```

Comment:
@mtokudome, I've changed these lines to `return "S1J6";` in my `impl`, following our Slack Chat.